### DISCUSSION TITLE
How to disable TOR

### DISCUSSION THREAD

**Discussion by marcoronk** [globaleaks/globaleaks-whistleblowing-software#3538] [Q&A]
Is it possible to disable Tor? In our company, the network does not allow you to use this service so we don't need it. In admin panel I couldn't do it. How is it possible?
Thanks.
Marco

**Comment by evilaliv3** [comment]
Yes, you can disable the "privacy panel" in the Advanced settings.

**Reply by esperti** [reply]
ok thanks but is it possible to do this selectively per channel and not globally for all channels?

**Reply by evilaliv3** [reply]
No actually. The privacy panel is a global widget visible on top of the entire application.

The Tor link though remains always active and available.
